frontend/awx/access/credentials/CredentialForm.tsx

```diff
@@ -249,31 +249,23 @@ function CredentialInputs({ isEditMode = false }: { isEditMode?: boolean }) {
         placeholder={t('Enter name')}
         isRequired
       />
-      <>
-        <PageFormSelect<Credential>
-          label={t('Credential type')}
-          isDisabled={isEditMode}
-          placeholderText={t('Select credential type')}
-          name="credential_type"
-          options={
-            itemsResponse?.results
-              .sort((l, r) => compareStrings(l.name, r.name))
-              .map((credentialType) => ({
-                label: credentialType.name,
-                value: credentialType.id,
-              })) ?? []
-          }
-          isRequired
-        />
-      </>
+      <PageFormTextInput<Credential>
+        name="description"
+        label={t('Description')}
+        placeholder={t('Enter description')}
+      />
       <PageFormSelectOrganization<Credential> isRequired={isGalaxyCredential} name="organization" />
-      <PageFormSection singleColumn>
-        <PageFormTextArea<Credential>
-          name="description"
-          label={t('Description')}
-          placeholder={t('Enter description')}
-        />
-      </PageFormSection>
+      <PageFormSelectCredentialType<Credential>
```

Comment:
Should the interface here be `CredentialType`?

Reply:
Yes, good catch.